### PR TITLE
Add basic docs for Docker, binary compiled to work w/o glibc

### PR DESCRIPTION
**Pull request by mattsimonsen**
The CGO_ENABLED flag was breaking compatibility with alpine linux, which does not support glibc.  

Please let me know if you need anything else to get this merged, the Docker file works great for me so hopefully the little documentation will help others.

### CODE REVIEW

File: hack/Dockerfile-goreleaser [hunk 1]
```diff
@@ -1,5 +0,0 @@
-FROM alpine:latest
```
Comment (mtulio) [inline hack/Dockerfile-goreleaser]: I need this Dockerfile to use with `goreleaser` tool and automate the cross build.

**Review by mtulio — CHANGES_REQUESTED** [review]

File: hack/Dockerfile-goreleaser (hunk 1 above)
Comment (mattsimonsen) [reply hack/Dockerfile-goreleaser]: I've merged to fix an updated documentation file, and added the second Dockerfile back.

**Review by mtulio — APPROVED** [review]
LGTM. Please rebase to merge.
Thanks

**Comment by mattsimonsen** [comment]
> LGTM. Please rebase to merge.
> Thanks


Apologies, please let me know if the rebase isn't as hoped.  

I believe my updated commit will apply cleanly.

**Review by mtulio — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -19,8 +19,8 @@ update:
 .PHONY: build
 build:
 	@test -d ./bin || mkdir ./bin
-	GOOS=$(GOOS) GOARCH=$(GOARCH) \
-		go build \
+	 CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH) \
```
Comment (mtulio) [inline Makefile]:
```suggestion
	CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH) \
```

File: Makefile [hunk 3]
```diff
@@ -19,8 +19,8 @@ update:
...
+		go build  \
```
Comment (mtulio) [inline Makefile]:
```suggestion
		go build \
```

File: Makefile-glob.mk [hunk 4]
```diff
@@ -25,6 +25,9 @@ GIT_DESCRIBE := $(shell git describe --tags --always)
 GOOS := linux
 GOARCH := amd64
 
+CGO_ENABLED := 0
+
+
```
Comment (mtulio) [inline Makefile-glob.mk]:
```suggestion
```

**Review by mtulio — CHANGES_REQUESTED** [review]
Hi @mattsimonsen , I've suggested small changes. :)
Thanks.

**Review by mtulio — APPROVED** [review]

**Comment by mtulio** [comment]
Thanks @mattsimonsen

**Pull request merged**